h2d/Mask.hx

```diff
@@ -1,7 +1,71 @@
 package h2d;
 
+private typedef RenderZoneStack = { hasRZ:Bool, x:Float, y:Float, w:Float, h:Float };
+
 class Mask extends Object {
 
+	static var renderZoneStack:Array<RenderZoneStack> = [];
+	static var renderZoneCaret:Int = 0;

```

Comment:
These should be moved to ctx

Reply:
Should I remake `RenderContext.setRenderZone` to `pushRenderZone` then?